### DISCUSSION TITLE
Using Tolgee with next-i18next

### DISCUSSION THREAD

**Discussion by datainvestor** [tolgee/tolgee-platform#1259] [Q&A]
I recently discovered Tolgee and I have been trying to implement in our teams App written in next.js that uses next-i18next for translations. I tried to find a guide for this integration but could not find any and I have been struggling to integrate Tolgee with next-i18next in next.js.

Anyone managed to do this and could share the correct approach?

**Comment by JanCizmar** [comment]
For next.js you can use React integration, which works fine.
